inspirehep/dojson/institutions/fields/bd1xx.py

```diff
@@ -139,6 +139,14 @@ def field_activity(self, key, value):
 @institutions.over('name_variants', '^410..')
 def name_variants(self, key, value):
     """Variants of the name."""
+    valid_sources = [
+        "DESY_AFF",
+        "ADS",
+        "INSPIRE"
+    ]
+    if value.get('9') and value.get('9') not in valid_sources:
+        return self.get('name_variants') if self.get('name_variants') else []
```

Comment:
But couldn't this be populated from before.? e.g. `902725` has more than one `name_variants`